File: HFSGuardaDiretorio_SharpDevelop_Ruby/comum/Log.rb
# 
# * Created by SharpDevelop.
# * User: henrique.souza
# * Date: 03/07/2015
# * Time: 15:10
# *
# * To change this template use Tools | Options | Coding | Edit Standard Headers.
# 
require "mscorlib"
require "System, Version=2.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a"

module HFSGuardaDiretorio.comum
	# <summary>
	# Description of Log.
	# </summary>
	class Log
		class MessageType
			def initialize()
				@Informational = 1
				@Failure = 2
				@Warning = 3
				@Error = 4
			end
		end

		def RecordMessage(Message, Severity)
		end

		def RecordMessage(Message, Severity)
		end
	end
end

File: HFSGuardaDiretorio_SharpDevelop_Python/comum/Log.py
# 
# * Created by SharpDevelop.
# * User: henrique.souza
# * Date: 03/07/2015
# * Time: 15:10
# *
# * To change this template use Tools | Options | Coding | Edit Standard Headers.
# 
from System import *

class Log(object):
	""" <summary>
	 Description of Log.
	 </summary>
	"""

	class MessageType(object):
		def __init__(self):
			self._Informational = 1
			self._Failure = 2
			self._Warning = 3
			self._Error = 4

	def RecordMessage(self, Message, Severity):
		pass

	def RecordMessage(self, Message, Severity):
		pass